tests/proxy/flake_id_generator/flake_id_generator_test.go

```diff
@@ -37,18 +38,30 @@ const (
 	idsInRoutine            = 100000
 )
 
+var remoteController RemoteController
+
 func TestMain(m *testing.M) {
-	remoteController, err := NewRemoteControllerClient("localhost:9701")
+	var err error
+	remoteController, err = NewRemoteControllerClient("localhost:9701")
 	if remoteController == nil || err != nil {
 		log.Fatal("create remote controller failed:", err)
 	}
-	cluster, err := remoteController.CreateCluster("3.9", DefaultServerConfig)
-	remoteController.StartMember(cluster.ID)
 	m.Run()
-	remoteController.ShutdownCluster(cluster.ID)
+}
+
+func asssignOverFlowId(clusterId string, instanceNum int) (r *Response, err error) {
+	script := "function assignOverflowedNodeId() {" +
+		"   instance_" + strconv.Itoa(instanceNum) + ".getCluster().getLocalMember().setMemberListJoinVersion(100000);" +
+		"   return instance_" + strconv.Itoa(instanceNum) + ".getCluster().getLocalMember().getMemberListJoinVersion();" +
+		"}" +
+		"result=\"\"+assignOverflowedNodeId();"
+	return remoteController.ExecuteOnController(clusterId, script, 1)
 }
 
 func TestFlakeIdGeneratorProxy_ConfigTest(t *testing.T) {
+	cluster, err := remoteController.CreateCluster("3.9", DefaultServerConfig)
```

Comment:
Does this mean cluster version is going to be 3.9? 3.9 does not even support flake ids.